src/FsAutoComplete.Core/TypedAstPatterns.fs

```diff
@@ -66,13 +66,8 @@ module SymbolUse =
         | _ -> None
 
     let (|StaticParameter|_|) (symbol : FSharpSymbolUse) =
-        #if NO_EXTENSIONTYPING
         Some
```

Comment:
is this legal code now?

Reply:
It's exactly the same code that has been compiling for netcoreapp2.1. Partial AP that always matches. 

Not really sure if that's *correct*, and *why* there were differences between Full Framework and Core.... but it's legal

Reply:
For posterity: this AP is never used, which is why we never had a problem with it.